### PR TITLE
API-15744  ../oas endpoint shouldn't retrieve all the OAS blobs

### PR DESCRIPTION
**Pull request by ediiotero**
Story: [API-15744](https://vajira.max.gov/browse/API-15744)

Added `lazy` property to `spec` and `transformedSpec` in the OASModel.

Now when using the  `GET: /docserver/apis/<apiName>/versions/<apiVersion>/oas` endpoint the OAS is removed to speed up the response.

Both `GET /docserver/apis/<apiName>/versions/<apiVersion>/oas/<oasIteration>` and
`GET /docserver/apis/<apiName>/versions/<apiVersion>/oas/latest` still work as expected

### CODE REVIEW

File: src/oas/oas.model.ts [hunk 1]
```diff
@@ -21,10 +21,10 @@ class OASModel extends BaseEntity {
...
 
-  @Property({ type: JsonType })
+  @Property({ type: JsonType, lazy: true })
   spec!: OAS;
 
-  @Property({ type: JsonType })
+  @Property({ type: JsonType, lazy: true })
   transformedSpec!: OAS;
 
   @Property({
```
Comment (christineyost) [inline src/oas/oas.model.ts:30]: Should this field also be lazy?

**Review by christineyost — COMMENTED** [review]
Looks good to me! Just wondering if the sf_transformed_spec should also be lazy?

File: src/oas/oas.model.ts (hunk 1 above)
Comment (ediiotero) [reply src/oas/oas.model.ts:30]: Yes, that makes sense.  Made the change.

**Review by mheusgit — APPROVED** [review]
Looking forward to testing this

**Pull request merged**